### ISSUE TITLE
[question] Does the request timeout includes connection time or not ?

### ISSUE THREAD

**Issue by tak2siva** [envoyproxy/envoy#3899]
*Description*:
Does the `request_timeout` in envoy includes `connection_time` or not ? Lets say I have `request_timeout` as 5s my upstream cluster connection takes more than 5s. Will I get req timeout or not ? (say I have 20s timeout for clusters so I wont get cluster_connection_timeout)

**Comment by mattklein123** [comment]
Yes, if a new upstream connection has to be made connection time is counted against the request timeout.
